components/txn_types/src/types.rs

```diff
@@ -247,6 +247,11 @@ impl Key {
     pub fn len(&self) -> usize {
         self.0.len()
     }
+
+    #[inline]
```

Comment:
I don't find where it's used. For `ObservedRange.to_range()` ?

Reply:
Yes.